xapian-glib/xapian-utils.cc

```diff
@@ -167,7 +181,8 @@ xapian_sortable_serialise (double value)
  * Since: 1.4
  */
 double
-xapian_sortable_unserialise (const char *value)
+xapian_sortable_unserialise (const guchar *value, gsize len)
```

Comment:
Coding style: arguments after the first go on a new line, aligned with the first argument.